rir/src/ir/insns.h

```diff
@@ -286,6 +291,13 @@ DEF_INSTR(aslogical_, 0, 1, 1, 0)
  */
 DEF_INSTR(asbool_, 0, 1, 1, 0)
 
+/**
+ * asint_ :: pop object stack, convert to integer scalar and push. Ceils/floors
+ *           if real, 0 or 1 if logical, throws an NA error if another type. For
```

Comment:
I would recommend creating two instructions. floor and ceil. that seems simpler and less confusing.